### ISSUE TITLE
[Usable] Tooltip does not defined for 'More' button.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1387]
**Actual Result:**
Tooltip does not defined for 'More' button.

**Expected Result:**
Tooltip should be define for 'More' button.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle](https://microsoft.github.io/BotFramework-WebChat/full-bundle)  in Edge browser.
2. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
3.  Navigate to "Sign In" button by using Tab key and press Enter on it to activate it.
4. Sign In page will appear on new Tab window.
5. Press F12 and open developer toolbar.
6. Check on 'More' button.
7. Observe that Tool tip is not define for 'more' button.

**User Impact:**
If user can not perceive the sensory characteristics of components such as shape, size, visual location, orientation, or sound, and no additional information is provided then the people with  disabilities and cognitive limitations will not be access the instructions for using the content.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** F12(Developer Toolbar)

**Attachment:**
![usable tooltip for more button](https://user-images.githubusercontent.com/41668935/48886567-70ae4c00-ee52-11e8-81c4-a274c5ccddcc.png)

**Comment by compulim** [comment]
This is not a bug from Web Chat. After sign-in button is clicked, the flow goes to Microsoft Account page.

Could you file this bug to the team that owns Microsoft Account page?

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with Accessibility team; this will be brought to Microsoft Account.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable

**Comment by pgopinadhu** [comment]
#A11yUsable;#A11ySev4;
